index.html

```diff
@@ -9,21 +9,25 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
-          <li>
-            <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
-            ></a>
+    <header class="header">
+      <nav class="navbar">
+        <a href="/" class="nav-logo">Training</a>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#projects" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about-me" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact" class="nav-link">Contact</a>
           </li>
         </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
```

Comment:
The project requirement requires to stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) (following the template that you chose).

**Consider fixing the following when you open the hamburger button:**
- [ ] Great job making your hamburger buttons, but they have so much space in between them so please remove the space between them and align them to the left instead of center(Check the picture as an example). 
**Your design choice:** 
![Screenshot from 2021-08-02 13-09-31](https://user-images.githubusercontent.com/62705947/127852774-184bb2a3-15d2-42dc-87e7-f7202d20469c.png))
**Your project:** 
![Screenshot from 2021-08-02 13-13-36](https://user-images.githubusercontent.com/62705947/127853152-94fdb878-6433-46a5-a294-38dd9e35a578.png)
- [ ] Currently flex-elements are in the center, align them to the left;
- [ ] When the hamburger button is open and I scroll down the options are still visible covering everything. When you scroll down with the hamburger button should not cover everything as you scroll down.